src/components/SearchForm.vue

```diff
@@ -1,22 +1,6 @@
 <template>
   <form class="search-form">
-    <input type="text" placeholder="Search" v-model="searchInput">
-    <article class="secondary-inputs">
-    <select name="genres" v-model="genreInput">
-      <option disabled>Genre:</option>
-      <option v-for="genre in genres" 
```

Comment:
Definitely think it makes sense to cut all of this since we aren't really using it